modules/onboard-azure-sql-managed-instance/outputs.tf

```diff
@@ -0,0 +1,35 @@
+output "default-managed-instance-security-rules" {
+  description = "Security rules required for Managed Instances by default."
+  value       = local.default_managed_instance_security_rules
+}
+
+output "security-group" {
+  description = "Azure Network Security Group."
+  value       = module.security-group.this
+}
+
+output "route-table" {
+  description = "Azure Managed Instance Route Table."
+  value       = module.route-table.this
+}
+
+output "virtual-network" {
+  description = "Azure Managed Instance Virtual Network."
+  value       = module.route-table.this
```

Comment:
typo `virtual-network`

Reply:
Am I blind? That's how you spell it, no?

Reply:
Not sure if you're being serious, just in case, I meant within the `value` field

Reply:
Okay so the answer to "am I blind" is apparently "yes" lol